RUNNING_IN_CONTAINERS.md

```diff
@@ -0,0 +1,62 @@
+# Running in Containers
+In case you want to run radicle in containers, on the same host (e.g. your laptop),
+you can use the `docker-compose.yml` file provided within this repo. 
+
+For the first node you want to start on your machine, you can: 
+
+## Create a profile 
+1. Create a folder where you will store the data of your node. e.g. `mkdir -p ~/radicle/profiles/bob/.radicle`
+1. Set `RAD_HOME` : `export RAD_HOME=~/radicle/profiles/bob/.radicle`
+1. Create a key:
+   - Pick a good passphrase and store it in your password manager
+   - go ahead with creating the key `rad auth --stdin` (or use `RAD_PASSPHRASE` env var)
+   - your profile should be created in `~/radicle/profiles/bob/.radicle`. 
+
+## Build the container images
+
+This takes a couple of minutes - depending on your machine - as it needs 
+to download the parent container images, and also all the rust 
+dependencies and then compile the code: 
+
+```bash
+docker-compose build
+```
+
+## Run First Node 
+1. Create a `.env.$nodename` file that will store all your environment variables:
+```yaml
+# these options are especially useful in a development setting - probably not for production use
+RADICLE_NODE_OPTIONS=--tracking-policy track --tracking-scope all
+# Note the difference between RAD_ROOT vs. RAD_HOME.
+RAD_ROOT=~/radicle/profiles/bob
+# ensure these ports are free on your machine
+RADICLE_API_PORT=8888
+RADICLE_NODE_PORT=8778
```

Comment:
Maybe we could rely on the defaults in this file to reduce the cognitive overhead and add a section with optional steps about overriding default port values in case there's a conflict?  Only a minority of users are going to need to configure that I imagine so we might as well sidetrack it.